src/blocks/GroupingBlock/GroupingBlock.astro

```diff
@@ -0,0 +1,61 @@
+---
+import type { GroupingBlockFragment } from '@lib/datocms/types';
+import Blocks from '@blocks/Blocks.astro';
+import { Accordion, AccordionItem } from '@components/Accordion';
+import { Tabs, TabsTab, TabsPanel } from '@components/Tabs';
+import type { AnyBlock } from '@blocks/Blocks';
+
+export interface Props {
+    block: GroupingBlockFragment
+}
+
+type GroupingItem = {
+  title: string;
+  blocks: AnyBlock[];
+};
```

Comment:
`GroupingItem` is only used inside GroupingBlock and does not exsist as a separate Block the content editor can add.

Reply:
Note that you do have a Fragment for this that you can import...